Compiler setting and installatoin
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,2 @@
+All %:
+	make $@ -C src
--- a/src/Makefile.inc
+++ b/src/Makefile.inc
@@ -37,20 +37,20 @@
 #ICON   = 
 
 ### for Linux, debian 6
-#QT4    =  /usr
-#CC     =  g++
-#CFLAGS = -O3 -Wall
-#INCS   = -I/usr/include/stlport -I/usr/include/qt4 -I/usr/X11R6/include
-#LIBS   = -lQtCore -lQtGui -lQtOpenGL -lQtXml -lGLU -lstlport -llapack -lblas
-#ICON   = 
+QT4    =  /usr
+CC     =  g++
+CFLAGS = -O3 -Wall
+INCS   = -I/usr/include/qt4 -I/usr/X11R6/include
+LIBS   = -lQtCore -lQtGui -lQtOpenGL -lQtXml -lGLU -llapack -lblas
+ICON   =
 
 ### for Mac
-QT4    =  $(HOME)/Qt-4.8.7
-CC     =  g++
-CFLAGS = -O3 -Wall -static
-INCS   = -I$(QT4)/include -I/usr/X11/include
-LIBS   = -F$(QT4)/lib -L$(QT4)/lib -lQtCore -lQtGui -lQtOpenGL -lQtXml -framework OpenGL -framework CoreFoundation -framework Foundation -framework CoreServices -framework Security -framework ApplicationServices -framework IOKit -framework Cocoa -framework Carbon -lz -llapack -lblas
-ICON   = 
+#QT4    =  $(HOME)/Qt-4.8.7
+#CC     =  g++
+#CFLAGS = -O3 -Wall -static
+#INCS   = -I$(QT4)/include -I/usr/X11/include
+#LIBS   = -F$(QT4)/lib -L$(QT4)/lib -lQtCore -lQtGui -lQtOpenGL -lQtXml -framework OpenGL -framework CoreFoundation -framework Foundation -framework CoreServices -framework Security -framework ApplicationServices -framework IOKit -framework Cocoa -framework Carbon -lz -llapack -lblas
+#ICON   =
 
 #########################################
 ### do not edit lines below
--- a/src/Makefile
+++ b/src/Makefile
@@ -57,7 +57,7 @@
 $(ICON): icon.rc icon.ico
 	windres.exe -i icon.rc -o icon.o
 
-clean:
+clean distclean:
 	rm -f *.moc.cpp *.exe *.exe.stackdump *.o *~ *.bak $(TARGET)
 
 tgz:
@@ -68,4 +68,10 @@
 	touch Makefile.dep
 	makedepend -fMakefile.dep -Y -- *.cc 2> /dev/null
 
+test: $(TARGET)
+
+install: $(TARGET)
+	install -d $(DESTDIR)/usr/libexec/tapioca
+	install $(TARGET) $(DESTDIR)/usr/libexec/tapioca
+
 include Makefile.dep
